treeherder/etl/perf.py

```diff
@@ -95,6 +95,9 @@ def _load_perf_artifact(project_name, reference_data, job_data, job_guid,
                        "load of performance artifacts".format(
                            perf_datum['framework']['name']))
         return
+    if perf_datum['framework']['enabled'] is False:
```

Comment:
This isn't the right place to look for enabled. You want the framework object that's a few lines above.